src/platform/site-wide/sass/consolidated-patches.scss

```diff
@@ -38,46 +38,65 @@ header.merger {
       background-image: none;
     }
   }
+
   @media (min-width: $medium-screen) {
-    va-notice--banner-inner, .va-crisis-line-container {
+
+    va-notice--banner-inner,
```

Comment:
It looks like this is missing a leading `.`, could you please add it?

Reply:
Sure.  Added the missing `.`.